compression/compression-including-empty-chunk.any.js

```diff
@@ -0,0 +1,57 @@
+// META: global=worker
+// META: script=pako/pako_inflate.min.js
+// META: timeout=long
+
+'use strict';
+
+// This test asserts that compressing '' doesn't affect the compressed data.
+// ex) compressing ['Hello', '', 'Hello'] results in 'HelloHello'
```

Comment:
I didn't realise at first that "ex)" means example. Could you write "Example:" instead?

Reply:
Okay! It may be Japanese style.